go.mod

```diff
@@ -5,8 +5,8 @@ go 1.18
 require (
 	github.com/Workiva/go-datastructures v1.0.53
 	github.com/bytecodealliance/wasmtime-go v0.39.0
+	github.com/wasmerio/wasmer-go v1.0.5-0.20220411075136-586bbfef6447
```

Comment:
ps this is a pre-release. it might be worth nagging wasmer to cut a release as they haven't in over a year, but probably would.

Reply:
@codefromthecrypt OK.  I'll give it a try to prod wasmer-go.  My pull was the last to land there and it has been a while.